docs/package_list.md

```diff
@@ -9,7 +9,7 @@
 | protocol/valory/ledger_api/1.0.0                              | `bafybeih7rhi5zvfvwakx5ifgxsz2cfipeecsh7bm3gnudjxtvhrygpcftq` |
 | connection/fetchai/http_server/0.22.0                         | `bafybeifbdjgzroxywqu5bimvj373yduhnc5n27vxbjf3fqs3mx7s6a5etq` |
 | connection/fetchai/stub/0.21.0                                | `bafybeieqlozydyvdxmjxhqygwq27djecpiftoqwlcpcr4qpotomwnh66yy` |
-| connection/valory/ledger/0.19.0                               | `bafybeic34ll726rlh4bwaeiuzmw5aoiw54ikvdgznlc5q2njv43affylpe` |
```

Comment:
Why the change?

Reply:
not sure, because of changes in loader.py? I never question the new hashes to be quite frank

Reply:
`loader.py` has nothing to with the hashes, please investigate this